base/atomics.jl

```diff
@@ -166,9 +263,20 @@ function atomic_xor! end
 
 Atomically store the maximum of `x` and `val` in `x`
 
-Performs `x[] = max(x[], val)` atomically. Returns the old (!) value.
+Performs `x[] = max(x[], val)` atomically. Returns the **old** value.
 
 For further details, see LLVM's `atomicrmw min` instruction.
```

Comment:
From what I can understand of the LLVM docs, this is just backwards. Will update